### DISCUSSION TITLE
problem with protoc - 3.5

### DISCUSSION THREAD

**Discussion by sleepwalker2017** [apache/incubator-mxnet#14175] [Q&A]
I downloaded the latest mxnet source code,and I noticed that it would download protoc-2.5 during compiling instead of using the lib on my PC.

But I must use protoc-3.5 in my application, so it just conflicts .

How can I solve this ? 

Thank you !

**Comment by ptrendx** [comment]
Are you trying to use version with distributed kvstore? In that case you should update you ps-lite submodule to include this commit: https://github.com/dmlc/ps-lite/commit/f45e2e78a7430be09f76264d2f4073fb2b1d54a2
